[{"body":"Install error:\r\n\r\n```\r\nTraceback (most recent call last):\r\n  File \"/srv/app/wsgi.py\", line 20, in <module>\r\n    application = make_app(config)\r\n  File \"/srv/app/src/ckan/ckan/config/middleware/__init__.py\", line 27, in make_app\r\n    load_environment(conf)\r\n  File \"/srv/app/src/ckan/ckan/config/environment.py\", line 69, in load_environment\r\n    p.load_all()\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 224, in load_all\r\n    load(*plugins)\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 240, in load\r\n    service = _get_service(plugin)\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 347, in _get_service\r\n    return plugin.load()(name=plugin_name)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 2442, in load\r\n    self.require(*args, **kwargs)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 2465, in require\r\n    items = working_set.resolve(reqs, env, installer, extras=self.extras)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 786, in resolve\r\n    raise DistributionNotFound(req, requirers)\r\npkg_resources.DistributionNotFound: The 'pkgutil-resolve-name>=1.3.10; python_version < \"3.9\"' distribution was not found and is required by jsonschema\r\nunable to load app 0 (mountpoint='') (callable not found or import error)\r\n*** no app loaded. going in full dynamic mode ***\r\n```\r\n\r\n[Related issue](https://github.com/transparencia-mg/work-stefanini/issues/95#issuecomment-1363333431)","comments":[{"id":"IC_kwDOIS9vFM5RaEiW","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"@gabrielbdornas eu tive este problema agora pela manhã refazendo o ambiente, fui instalando cada dependência que indicava pendência até voltar ao problema do dataclasses, mas encontrei uma possível explicação:\r\n\r\n[Indicação da remoção do dataclasses como dependência - 10/10/2022](https://github.com/Textualize/rich/blob/234d1eaa2a430c7a5766890e3bec13befca6dbc4/poetry.lockl)\r\n\r\n![Última versão lançada da biblioteca rich 02/10/2022](https://user-images.githubusercontent.com/223948/209652042-7485efaf-cc2e-432c-8cd6-29cf5ed345f3.png)\r\n\r\nA tag 12.6.0 que é justamente da última versão instalada da rich contém essa dependência da dataclasses:\r\n\r\n![Arquivo do poetry.lock](https://user-images.githubusercontent.com/223948/209652434-6e39702b-0e8a-45aa-9545-31b2975dce44.png)\r\n","createdAt":"2022-12-27T10:23:12Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/8#issuecomment-1365788822","viewerDidAuthor":false}],"title":"Version 0.0.7 intall error"},{"body":"Error during package installation:\r\n\r\n```\r\nTraceback (most recent call last):\r\n  File \"/srv/app/wsgi.py\", line 20, in <module>\r\n    application = make_app(config)\r\n  File \"/srv/app/src/ckan/ckan/config/middleware/__init__.py\", line 27, in make_app\r\n    load_environment(conf)\r\n  File \"/srv/app/src/ckan/ckan/config/environment.py\", line 69, in load_environment\r\n    p.load_all()\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 224, in load_all\r\n    load(*plugins)\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 240, in load\r\n    service = _get_service(plugin)\r\n  File \"/srv/app/src/ckan/ckan/plugins/core.py\", line 347, in _get_service\r\n    return plugin.load()(name=plugin_name)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 2442, in load\r\n    self.require(*args, **kwargs)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 2465, in require\r\n    items = working_set.resolve(reqs, env, installer, extras=self.extras)\r\n  File \"/usr/lib/python3.10/site-packages/pkg_resources/__init__.py\", line 786, in resolve\r\n    raise DistributionNotFound(req, requirers)\r\npkg_resources.DistributionNotFound: The 'pkgutil-resolve-name>=1.3.10; python_version < \"3.9\"' distribution was not found and is required by jsonschema\r\nunable to load app 0 (mountpoint='') (callable not found or import error)\r\n*** no app loaded. going in full dynamic mode ***\r\n```\r\n\r\n[Issue related](https://github.com/transparencia-mg/work-stefanini/issues/95#issuecomment-1363333431).","comments":[],"title":"Version 0.0.6 Install error "},{"body":"@gabrielbdornas, @andreiadonca e @Andrelamor foram adicionadas as instruções de instalação no README.md e publicada a nova versão","comments":[{"id":"IC_kwDOIS9vFM5RITBc","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"@gileno , nestes termos fecharei o issue! ","createdAt":"2022-12-21T10:23:53Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/5#issuecomment-1361129564","viewerDidAuthor":false}],"title":"Instruções para instalação"},{"body":"@gabrielbdornas, @andreiadonca e @Andrelamor a extensão `ckanext validation` cria um novo modelo/tabela para salvamento das validações como visto em:\r\n\r\n- [models.py](https://github.com/frictionlessdata/ckanext-validation/blob/master/ckanext/validation/model.py)\r\n\r\nSeguindo a ideia deles seria melhor ter um modelo / tabela para salvar os metadados ao invés de salvar nos extras, vocês acham que o fato deles não salvarem nos extras faz com que seja melhor não salvarmos nos extras? Falta apenas essa decisão de tomada para subir as telas atualizadas salvando corretamente os dados\r\n","comments":[{"id":"IC_kwDOIS9vFM5RIUXa","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"@gileno , conforme informado por você a mudança para salvar em um novo modelo já foi realizada,  sendo assim fecharei o issue. ","createdAt":"2022-12-21T10:28:59Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/4#issuecomment-1361135066","viewerDidAuthor":false}],"title":"Salvamento dos metadados"},{"body":"@gileno do you mand if we change our branch `master` to `main`?","comments":[{"id":"IC_kwDOIS9vFM5P2zji","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"@gabrielbdornas I didn't realize it was master, I believe it was because the repository was created if the README.md and initial .gitignore\r\nI will do","createdAt":"2022-12-06T18:04:00Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1339767010","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5P2z2D","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"> @gabrielbdornas I didn't realize it was master, I believe it was because the repository was created if the README.md and initial .gitignore I will do\r\n\r\nGreat!","createdAt":"2022-12-06T18:04:47Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1339768195","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5P9aP6","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"@gabrielbdornas I created the branch \"main\" but it is necessary to go to the repository settings to change it on github","createdAt":"2022-12-07T19:42:27Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1341498362","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5RI2V2","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"@gileno, modifiquei a branch principal do projeto para main:\r\n\r\n![image](https://user-images.githubusercontent.com/49699290/208909192-ae4fbc0e-3770-4a84-9a4b-f2c2a377fe8e.png)\r\n\r\nSolicito que realize o merge de tudo que está na master (ATUALIZAR COM EVENTUAIS COMMITS DA SUA MÁQUINA QUE POR VENTURA NÃO TENHA SIDO SINCRONIZADOS VIA PULL) na main via criação e aprovação de um PR.\r\n\r\nTambém habilitei esta funcionalidade no repositório:\r\n\r\n![image](https://user-images.githubusercontent.com/49699290/208909557-d4a5b5ee-c786-4a8b-b644-1975326022d0.png)\r\n\r\nSendo assim, quando você aprovar o PR solicitado acima a branch `master` será excluída automanticamente. Após todo processo recomendo exclui-la de sua máquina também (`git branch -D master`) para evitar confusões.\r\n\r\nEm caso de dúvidas estou à disposição.\r\n","createdAt":"2022-12-21T12:49:37Z","includesCreatedEdit":true,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1361274230","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5RJCKp","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"@gabrielbdornas irei verificar","createdAt":"2022-12-21T13:37:22Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1361322665","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5RXysm","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"Issue resolvido dia 26/12/2022 em reunião na parte da manhã com @gileno ","createdAt":"2022-12-26T13:35:54Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/3#issuecomment-1365191462","viewerDidAuthor":false}],"title":"Atualizar branch principal do projeto para `main`"},{"body":"@gileno as talked, we must create an automated process to publish/update the package on [pypi](https://pypi.org/).\r\n\r\nWe did this in our [dpckan](https://github.com/transparencia-mg/dpckan) package using [this `Makefile`](https://github.com/transparencia-mg/dpckan/blob/dev/Makefile). We could do the same here.\r\n\r\nThe only problem/inconvenient will be updating the package version on `setup.py` file every new update on  [pypi](https://pypi.org/) because it doesn't allow to publish new code in already publish versions.\r\n\r\nhttps://github.com/transparencia-mg/ckanext-datapackage-creator/blob/b7c1e718885dbdeb0c159c1c2b5534e2d551c45c/setup.py#L8-L11\r\n\r\nTo better document these new releases we could use our brand new [CHANGLOG.md](https://github.com/transparencia-mg/ckanext-datapackage-creator/blob/master/CHANGELOG.md) file.\r\n\r\nTo finish the task, please document all the steps used to realize the update in the `README.md` file. [Here is an example](https://github.com/transparencia-mg/dpckan/issues/6#issuecomment-851678297) that could be followed. \r\n\r\nTODO:\r\n- [x] Create the `Makefile` and its necessary targets.\r\n- [x] Publish package on PyPI (DTA account).\r\n- [x] Include @gileno as \"package maintainer\"\r\n- [x] Include @gabrielbdornas as package maintainer\"\r\n- [x] Document process on `README.md` file.","comments":[{"id":"IC_kwDOIS9vFM5Phmr6","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"Ok, I will update","createdAt":"2022-12-01T18:57:11Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1334209274","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5PhtDL","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"@gileno what's your Pypi user?","createdAt":"2022-12-01T19:18:40Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1334235339","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5PhxEe","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"Just **gileno** ","createdAt":"2022-12-01T19:28:13Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1334251806","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5P2rEe","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"> Just **gileno**\r\n\r\n@gileno you've been invited as package `maintainer` in [pypi](https://pypi.org/project/ckanext-datapackage-creator/)\r\n\r\n![image](https://user-images.githubusercontent.com/49699290/205981551-c9311b79-8cf6-4f88-b03a-9e17c6c8aa5c.png)\r\n","createdAt":"2022-12-06T17:33:18Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1339732254","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5P2sgL","author":{"login":"gileno"},"authorAssociation":"MEMBER","body":"@gabrielbdornas I accepted the invitation","createdAt":"2022-12-06T17:38:04Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1339738123","viewerDidAuthor":false},{"id":"IC_kwDOIS9vFM5P2w3c","author":{"login":"gabrielbdornas"},"authorAssociation":"MEMBER","body":"> @gabrielbdornas I accepted the invitation\r\n\r\n@gileno fine. The Package was published successfully. And all process was documented in [`README.md` file](https://github.com/transparencia-mg/ckanext-datapackage-creator). ","createdAt":"2022-12-06T17:54:34Z","includesCreatedEdit":false,"isMinimized":false,"minimizedReason":"","reactionGroups":[],"url":"https://github.com/transparencia-mg/ckanext-datapackage-creator/issues/1#issuecomment-1339755996","viewerDidAuthor":false}],"title":"Create process to publish/update package on pypi"}]
